magento/magento2
# Added 'clearEvent' to eventBus  to prevent multiple attach event on wysiwyg editor

### Description (*)


### Fixed Issues (if relevant)

1. https://github.com/magento/adobe-stock-integration/issues/689

### Manual testing scenarios (*)
set `console.log('openfileBrowser')` on 376 line in  `lib/web/mage/adminhtml/wysiwyg/tiny_mce/tinymce4Adapter.js`
1.1 open browser dev tools
1. Open Catalog -> product -> Add new product
2. Content Tab -> on wisiwyg editor press image icon -> source click folder button

**Expected result** openfileBrowser appears one time on console dev tools when you open file bwroser

[Case2]
AbodeStock 
Open Adobe Stock on product page in content tab 
**Expected result** Adobe Stock button opens grid
 ### Questions or comments


### Contribution checklist (*)
